src/remote.sh

```diff
@@ -222,40 +224,51 @@ function generate_tarball()
   fi
 }
 
-# Handles the remote info
+# Populate remote info
 #
-# @parameters String to be parsed
+# @parameters: Command line parameter to be parsed
 #
 # Returns:
-# This function has two returns, and we make the second return by using
-# capturing the "echo" output. The standard return ("$?") can be 22 if
-# something is wrong or 0 if everything finished as expected; the second
-# output is the remote info as IP:PORT
-function get_remote_info()
+# This function populates the variables REMOTE_IP and REMOTE_PORT based on the
```

Comment:
Maybe specifying that they are entries of the remote_parameters array would clear things a bit. I propose:

`This function populates the entries REMOTE_IP and REMOTE_PORT of the remote_parameters array...`